www/src/pages/table/NodeSearchComponent.tsx

```diff
@@ -128,19 +128,13 @@ export const NodeSearchComponent = withStyles(styles)(
         classStore
           .fetchSearchResults(query)
           .then((data) => {
-            data.forEach((searchResult: ClassTextSearchResult) => {
```

Comment:
rename it to ClassTextSearchResult, as we are using same naming convention for Entity and properties.